core/src/test/java/org/apache/iceberg/rest/auth/AuthSessionUtil.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.rest.auth;
+
+import org.apache.iceberg.rest.auth.OAuth2Util.AuthSession;
+
+/** Helper class to make the token refresh retries configurable for testing */
+public class AuthSessionUtil {
+
+  private AuthSessionUtil() {}
+
+  public static void withTokenRefreshNumRetries(int retries) {
+    AuthSession.withTokenRefreshNumRetries(retries);
```

Comment:
I don't think `with` makes sense here. Can you change these to `set`?

Reply:
done